app/src/main/java/com/simplecity/amp_library/ui/screens/nowplaying/PlayerPresenter.kt

```diff
@@ -302,8 +303,9 @@ class PlayerPresenter @Inject constructor(
     }
 
     fun saveUiEvent(uiEventType: UiEventType){
-        if (PlayerFragment.getSong() != null) {
-            val uiEvent = EventUtils.newUiEvent(PlayerFragment.getSong(), uiEventType, context)
+        val song = MusicServiceConnectionUtils.getSong()
+        if (song != null) {
+            val uiEvent = EventUtils.newUiEvent(song, uiEventType, context)
```

Comment:
Same for all cases where logging events - remove null check

Reply:
Same here. AS throws an error, because the argument type is `song` instead of `song?`

Reply:
I think this is a good use case for `song!!`, assuming that we believe that `song` should never be `null` here.